webapp/platform/components/src/index.tsx

```diff
@@ -2,13 +2,14 @@
 // See LICENSE.txt for license information.
 
 // type
-export type {Props as GenericModalProps} from './generic_modal/generic_modal';
+export type {Props as LegacyGenericModalProps} from './legacy_generic_modal/legacy_generic_modal';
 export type {CircleSkeletonLoaderProps, RectangleSkeletonLoaderProps} from './skeleton_loader';
 export type {Props as FocusTrapProps} from './focus_trap';
 
 // components
-export * from './generic_modal/footer_content';
-export {GenericModal} from './generic_modal/generic_modal';
+export * from './compass-ui';
```

Comment:
Nothing to change here for now, but this is already starting to make me question exporting all of these components as a single module. The module's not particularly big yet, but it doesn't support code splitting, so it might start to cause the bundle size to grow a lot once more components are added